### PR TITLE
feat(npm)!: remove RENOVATE_CACHE_NPM_MINUTES 

### PR DESCRIPTION
**Pull request by rarkins**
## Changes

Remove support for RENOVATE_CACHE_NPM_MINUTES

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/util/package-rules/package-patterns.spec.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ describe('util/package-rules/package-patterns', () => {
     });
 
     it('should fall back to matching depName', () => {
-      const result = packagePatternsMatcher.matches(
+      const result = packageNameMatcher.matches(
```
Comment (viceice) [inline lib/util/package-rules/package-patterns.spec.ts:32]: wrong change?

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: lib/util/package-rules/package-patterns.spec.ts (hunk 1 above)
Comment (rarkins) [inline lib/util/package-rules/package-patterns.spec.ts:32]:
```suggestion
      const result = packagePatternsMatcher.matches(
```

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
it seems we've failing tests on v38?

**Pull request merged**